learning-outcomes.md

```diff
@@ -0,0 +1,38 @@
+# Week 2 Learning Outcomes
+
+## HTTP
+
+### Asynchronicity
+
+- [ ] **We can write code that executes asynchronously**
+- [ ] We can use callbacks to make our code non blocking
+
+### Fetch
+
+- [ ] **We can use the fetch method to make API requests and receive responses**
```

Comment:
```suggestion
- [ ] **We can use the fetch method to make HTTP requests and receive responses**
```
"API" is used for so much stuff it gets confusing. Also you can make requests to servers that wouldn't necessarily be what most think of as an API